pfcpiface/telemetry.go

```diff
@@ -96,6 +97,12 @@ func (uc *upfCollector) Describe(ch chan<- *prometheus.Desc) {
 
 	ch <- uc.latency
 	ch <- uc.jitter
+
+	ch <- uc.sessionLatency
+	ch <- uc.sessionJitter
+	ch <- uc.sessionTxPackets
+	ch <- uc.sessionDroppedPackets
+	ch <- uc.sessionTxBytes
```

Comment:
ideally we should have separate interface for these, like we have one for PFCP layer metrics. 

https://github.com/omec-project/upf-epc/blob/1ae479b109b9dfc62e341f0233af20d40e2c5d74/pfcpiface/metrics/interface.go#L48-L51

Overall split could be -

1. PFCP protocol metrics - interface - how many messages, sessions etc
2. PFCP fastpath metrics - interface - fastpath performance of the session/pdr, can benefit from more PFCP context
3. Fastpath backend specific metrics - need not be interface, registered/exported in their own files - fastpath wide latency. jitter, packets, bytes, dropped, memory util etc